CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ else ()
   endif(WIN32)
 endif ()
 
-option(USE_BLAS "Compile cminpack using a blas library if possible" ON)
+option(USE_CBLAS "Compile cminpack using cblas library if possible" ON)
```

Comment:
I would rather not rename the option, because this may break packages. Can you please revert the option name to USE_BLAS (leave cblas in the function documentation).

Reply:
and the doc should say "using *a* cblas library" (there may be several cblas libraries installed)